tests/integration/utility_test_functions.py

```diff
@@ -32,3 +36,86 @@ def run_cromshell_command(command: list, exit_code: int):
             f"\n{print_exception(*result.exc_info)}"
         )
         return result
+
+
+def wait_for_workflow_completion(test_workflow_id: str):
+    import time
+
+    count = 0
+    status = ""
+    print("Printing workflow status:")
+    while count < 10:
+        time.sleep(5)
+        count += 1
+        # run status check every 5 sec until done, max 10x
+        status_result = run_cromshell_command(
+            command=["status", test_workflow_id],
+            exit_code=0,
+        )
+        print(status_result.stdout)
+        status_result_formatted = json.loads(status_result.stdout)
+        status = status_result_formatted["status"]
+
+        if status == "Failed" or status == "Succeeded":
```

Comment:
"Doom" does occur but in this test function it is considered to be nonterminal. the loop will continue until the workflow fails or it reaches the 50 seconds at which point the test will fail for still being in "Doom" status. 
This will need to be in a different PR but it would be good to have a workflow specifically for this situation such that it is in "Doom" status long enough for the test can repeatedly catch it.

Reply:
For that future PR, may be the task could be simply `sleep 60`, that'd emulate a task that's longer than 50 seconds.